pandas-stubs/_testing/__init__.pyi

```diff
@@ -48,11 +42,7 @@ def assert_numpy_array_equal(
     obj: str = ...,
 ): ...
 def assert_extension_array_equal(
-    left,
-    right,
-    check_dtype: bool = ...,
-    check_less_precise: bool = ...,
-    check_exact: bool = ...,
+    left, right, check_dtype: bool = ..., check_exact: bool = ...
```

Comment:
```suggestion
    left, right, check_dtype: bool = ..., index_values: Index | np.ndarray | None = ..., check_exact: bool = ..., 
    rtol: float = ..., atol: float = ..., obj: str = ...
```

Looking at the docs, the arguments were previously wrong

Reply:
@Dr-Irv I added all documented parameters.